packages/server/lib/util/ci_provider.js

```diff
@@ -282,7 +282,8 @@ const _providerCommitParams = function () {
   return {
     appveyor: {
       sha: env.APPVEYOR_REPO_COMMIT,
-      branch: env.APPVEYOR_REPO_BRANCH,
+      // since APPVEYOR_REPO_BRANCH will be the target branch on a PR
```

Comment:
I would add comment with forked PR example vs standard PR

Reply:
there should be no difference, only APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME will change with forked PRs